gordo_components/builder/build_model.py

```diff
@@ -45,10 +46,13 @@ def build_model(
         Mapping of the Dataset to initialize, following the same logic as model_config.
     metadata: dict
         Mapping of arbitrary metadata data.
+    cv_mode: 
+        String which enables three different modes: Only printing cv_scores after cross validation, build the model i.e.
+        no cross validation or do a full build. Default is full build with cross validation.
```

Comment:
you explain the different modes, but not what values they should be for each. If we don't go with the `Enum` option, we should at least list each option out here with bullet points or something to that effect.